README.md

```diff
@@ -47,3 +47,5 @@ make build verify
 ```bash
 make image
 ```
+
+- To test your modified CSI driver operator on a test cluster follow [Docs](docs/testing-custom-operator.md)
```

Comment:
Can you also removed the `manifests` directory? Example: https://github.com/openshift/aws-ebs-csi-driver-operator/pull/127/files

Reply:
I'll do it in a separate PR

Reply:
I think that someone forgot about us :smiley: :
![image](https://user-images.githubusercontent.com/38602062/122934484-3b569b80-d378-11eb-95c1-72e83757a1a7.png)